crontab

```diff
@@ -7,5 +7,5 @@ SHELL=/bin/bash
 #
 #  SDO Analysis to fetch from IETF, YangModels, ...
 #
-5 1 * * 0 (cd /sdo_analysis/bin/ ; echo "`date` starting cronjob-weekly" > /var/yang/logs/crons-log.log ; ./cronjob-weekly) > /var/yang/logs/cronjob-weekly.log
-5 1 * * 1-6 (cd /sdo_analysis/bin/ ; echo "`date` starting cronjob-daily" > /var/yang/logs/crons-log.log ; ./cronjob-daily) > /var/yang/logs/cronjob-daily.log
+5 1 * * 6 (cd /sdo_analysis/bin/ ; echo "`date` starting cronjob-weekly" > /var/yang/logs/crons-log.log ; ./cronjob-weekly) > /var/yang/logs/cronjob-weekly.log
+5 1 * * 0-5 (cd /sdo_analysis/bin/ ; echo "`date` starting cronjob-daily" > /var/yang/logs/crons-log.log ; ./cronjob-daily) > /var/yang/logs/cronjob-daily.log
```

Comment:
0 means sunday I would still keep this to start from every monday to friday or saturday

Reply:
Fixed: cronjob-daily will run every day except Sunday - which means that nothing changes for cronjob-daily